package.json

```diff
@@ -390,8 +390,8 @@
     "test-unit-watch": "yarn test-unit --watch",
     "test-unit-watch:cljs": "yarn concurrently -n 'cljs,tests' 'yarn build-watch:cljs' 'yarn test-unit-keep-cljs --watch'",
     "test-visual": "yarn build && ./bin/build-for-test && yarn test-visual-run",
-    "test-visual:loki": "yarn loki test",
-    "test-visual:loki-update": "yarn loki update",
+    "test-visual:loki": "yarn loki test --chromeFlags='--headless --disable-gpu'",
```

Comment:
Loki's default `chromeFlags` option includes the `--hide-scrollbars` flag ([docs](https://loki.js.org/command-line-arguments.html)), which we don't want.